### ISSUE TITLE
Issues running in Centos 5

### ISSUE THREAD

**Issue by teropes** [strongloop/loopback-connector-oracle#50]
I'm having issues starting loopback server with loopback-connector-oracle in Centos 5 32-bit environment. During server start I get error message about loopback connector "oracle" not being installed. During npm install there are no error messages. I am sure that I have sourced ~/strong-oracle.rc and LD_LIBRARY_PATH looks correct.

I have done several clean minimal installations of Centos 5 and Centos 6 with bare minimum to run our loopback server software: gcc, python 2.7 and node v0.12.7. Each time Centos 6 works without any issues and Centos 5 fails to start. I even installed the node-oracledb low level driver and some other modules that require compiling and they seem to work just fine so there should not be anything totally wrong in my dev environment.

My question is: is there something in the loopback-connector-oracle or strong-oracle that maybe requires some system level packages that does not exist / are too old in Centos 5? The error message is so vague, it just states that its not installed when it actually is. However the oracledb driver and oracle database installation itself is supported even in RHEL5 so weird that the wrapper wouldnt be.

**Comment by superkhau** [comment]
The driver uses the official oracle driver underneath the hood so I don't see why it wouldn't work exactly the same. @raymondfeng Any ideas?

@cgole Do you have Centos 5 and 6 VMs? Can you help reproduce this to confirm the issue?

**Comment by teropes** [comment]
Here is some extra information if it helps:
I found a workaround so I can at least get it to work now:
1. clone server from git and npm install => installs ok, no npm-debug.log or any errors, still it does not work
2. go manually into servers node_modules/loopback-connector-oracle/node_modules/strong-oracle
3. node-gyp configure
4. node-gyp rebuild

I can now get it to work but I dont understand the why it actually fixes the problem. I looked at the strong-oracles build/Release folder and it had oracle_bindings.node after npm install.

This happens every time on my Centos 5, on Centos 6 just the npm install is enough. In both cases I have made sure I have sourced my strong-oracle.rc and LD_LIBRARY_PATH looks correct.

My deps in servers package.json looks like this:
  "dependencies": {
    "compression": "^1.0.3",
    "errorhandler": "^1.1.1",
    "loopback": "^2.8.0",
    "loopback-boot": "^2.4.0",
    "loopback-connector-oracle": "^1.5.0",
    "loopback-datasource-juggler": "^2.7.0",
    "serve-favicon": "^2.0.1",
    "winston": "^2.0.1"
  }
